### ISSUE TITLE
Keyboard focus indicator is not clearly visible on page controls.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#123]
User Experience:

For keyboard users, it is hard to recognize the element which contains current focus if visual focus indicator is not visible on any controls.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Pages button.

5. Navigate to the various controls of the "Pages" window.

6. Verify the keyboard focus visible clearly or not on all controls.

 

Actual Result:

Keyboard focus indicator is not clearly visible on tab controls as ‘Pages, Media…etc..’. and controls present inside tab. Only half rectangle line visible on all controls.

 

Expected Result:

Keyboard focus indicator should be clearly visible on tab controls as ‘Pages, Media…etc..’., and controls present inside tab. Full rectangle lines should be visible on all controls. So that user can identify that which element contains current focus.

 

Note:

Similar issue exist with Chrome.

Similar issue exist with all controls present in application.

 

Reference Links

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
